src/Components/CreateImageWizard/steps/Oscap/index.tsx

```diff
@@ -154,9 +154,9 @@ const OscapContent = () => {
         </Title>
         <Content>
           Select which {lightspeedEnabled ? 'Red Hat Lightspeed' : 'Insights'}{' '}
-          Compliance policy or OpenSCAP profile you want your image to be
+          compliance policy or OpenSCAP profile you want your image to be
           compliant-ready for.{' '}
-          {lightspeedEnabled ? 'Red Hat Lightspeed' : 'Insights'} Compliance
+          {lightspeedEnabled ? 'Red Hat Lightspeed' : 'Insights'} compliance
```

Comment:
This is missing a space, I think.
You may have to add `{' '}` before compliance not to get it mangled into Lightspeed